component-archetype/src/main/resources/archetype-resources/.gitignore

```diff
@@ -0,0 +1,54 @@
+/target/
+
+.cache-main
+.cache-tests
+
+# use glob syntax.
+syntax: glob
+*.ser
+*.class
+*~
+*.bak
+#*.off
+*.old
+
+# eclipse conf file
+.settings
+.classpath
+.manager
+.scala_dependencies
+
+# idea
+.idea
+*.iml
+
+# building
+target
+build
+null
+tmp*
+temp*
+dist
+test-output
+build.log
+
+# other scm
+.svn
+.CVS
+.hg*
+
+# switch to regexp syntax.
+#  syntax: regexp
+#  ^\.pc/
+
+#SHITTY output not in target directory
```

Comment:
**#SHITTY** true but politically incorrect. Maybe remove also from main .gitignore? Sorry, my fault.

Reply:
ok will remove it :D